Readme.md

```diff
@@ -17,9 +17,18 @@ interactions without compromising either performance or ease of use.
 npm i capi
 ```
 
-> Note: The minimum supported Node version is 20, as we require the
+> Note: The minimum supported Node version is 20.3.1. We prioritize browser
```

Comment:
While important, this is somewhat cluttering the readme's beginning. Anything we can do to make it less intrusive? Can we try placing within a detail element, with the text "Using with Node 20.3 and below" ?

Reply:
I suggest:

`> Note: The minimum supported Node version is 20.3.1, as Capi requires the standard Web Crypto API.`

Reply:
Agreed.

Let's include that shim line in the docs' setup page.

```ts
globalThis.crypto = require("node:crypto").webcrypto
```

We can link to it from an additional sentence of this note: `See instructions for shimming in older versions of Node.`

Thoughts?

Reply:
Do we like it as a details block here?

https://github.com/paritytech/docs.capi.dev/pull/40

Reply:
Yes we do! We can link to it as https://docs.capi.dev/setup/#requirements (updated your PR to have a section)

Reply:
Just pushed the following. Any tweaks we should make?

<img width="448" alt="Screenshot 2023-07-13 at 8 11 16 PM" src="https://github.com/paritytech/capi/assets/4893548/c5305fd2-faa7-4218-bf17-c169bcba8dd6">